dashboard/src/index.tsx

```diff
@@ -7,19 +7,10 @@ import { axios } from "./shared/Auth";
 import Root from "./containers/Root";
 import "./index.css";
 import store from "./store";
-// import registerServiceWorker from "./registerServiceWorker";
 
 createAxiosInterceptors(axios, store);
 
 ReactDOM.render(<Root />, document.getElementById("root") as HTMLElement);
 
-// Disable the service worker as it currently loads index.html from cache for
```

Comment:
it seemed to me that this comment was outdated, I couldn't find any reference to `registerServiceWorker`